ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/hooks/useWatchLastSnapshotForComplianceReports.ts

```diff
@@ -19,7 +19,7 @@ async function fetchLastJobForConfiguration(
         perPage: 1,
         showMyHistory: true,
         sortOption: {
-            field: 'Report Completion Time',
+            field: 'Compliance Report Completed Time',
```

Comment:
The filters and sorting terms are a bit different with Compliance Reporting. Making changes for those.